loader/checkpoint.go

```diff
@@ -125,19 +131,19 @@ func (cp *RemoteCheckPoint) createTable() error {
 	);
 `
 	sql2 := fmt.Sprintf(createTable, tableName)
-	err := cp.conn.executeSQL([]string{sql2}, true)
+	err := cp.conn.executeSQL(cp.tctx, []string{sql2}, true)
 	return errors.Trace(err)
 }
 
 // Load implements CheckPoint.Load
 func (cp *RemoteCheckPoint) Load() error {
 	begin := time.Now()
 	defer func() {
-		log.Infof("[checkpoint] load checkpoint takes %f seconds", time.Since(begin).Seconds())
+		cp.tctx.L().Info("load checkpoint", zap.Float64("cost time", time.Since(begin).Seconds()))
```

Comment:
we can refine all cost time use scenario to `zap.Stringer("cost time", time.Since(begin))`, which will automatically generate time unit

Reply:
Use `zap.Duration` instead of `zap.Stringer`...